javaslang/src/main/java/javaslang/Lazy.java

```diff
@@ -94,7 +94,7 @@ private Lazy(Supplier<? extends T> supplier) {
      */
     public static <T> Lazy<Seq<T>> sequence(Iterable<? extends Lazy<? extends T>> values) {
         Objects.requireNonNull(values, "values is null");
-        return Lazy.of(() -> List.ofAll(values).map(Lazy::get));
+        return Lazy.of(() -> List.ofAll(values).map(l -> l.get()));
```

Comment:
I kinda' agree, the build isn't passing now either. I already opened a bug for a compiler crash, but they didn't fix it yet, I had to circumvent it.
When we do have an error though (please see the Travis log or pull it locally) or a simplification (e.g. the `Character` constructor), we should definitely take advantage of Java 9's wisdom.